float32 ref1
float32 ref2
float32 ref3
---
geometry_msgs/Vector3 angles